### PR TITLE
updated the wrong categories

### PR DESCRIPTION
**Pull request by edewit**
fixed [AGCORDOVA-112](https://issues.jboss.org/browse/AGCORDOVA-112)

### CODE REVIEW

**Comment by secondsun** [comment]
Thanks man.
1 : Can you add an automated test?
2 : Can you also send this PR to the 2.2.x branch?  passos and I can wallop it and push an update quicker.

**Comment by edewit** [comment]
on it

**Comment by secondsun** [comment]
Oh yeah also can you give AeroGearGCMPushJsonConfiguration the same treatment?

**Comment by danielpassos** [comment]
Could you also rebase? I've just merge a PR fixing the travis build

File: aerogear-android-push-test/src/main/java/org/jboss/aerogear/android/unifiedpush/test/gcm/UnifiedPushConfigTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package org.jboss.aerogear.android.unifiedpush.test.gcm;
```
Comment (secondsun) [inline aerogear-android-push-test/src/main/java/org/jboss/aerogear/android/unifiedpush/test/gcm/UnifiedPushConfigTest.java]: License Headers

**Comment by secondsun** [comment]
Everything seems to be working for me.  Test pass and I tested with the hello world app and several permutations of categories.

**Comment by edewit** [comment]
@danielpassos I do setCategories twice, because the 'old' implementation did an add instead of set. So performing the set twice would break it.

**Comment by secondsun** [comment]
@edewit Ahhh interesting.  Can you put that comment in the test and then I think we are good to merge.

**Comment by danielpassos** [comment]
:+1:

**Pull request merged**